feat: Prefer local image (#381)
project-copacetic/copacetic: website/docs/quick-start.md

* An Ubuntu 22.04 VM configured through the [setup instructions](./installation.md). This includes:
  * `copa` tool [built & pathed](./installation.md).
  * [buildkit](https://github.com/moby/buildkit/#quick-start) daemon installed & pathed. [Examples](#buildkit-connection-examples)
    * The `docker` daemon runs a buildkit service in-process. If you are using this for your buildkit instance, Docker must have the

>       [containerd snapshotter feature](https://docs.docker.com/storage/containerd/) enabled.

Reviewer: ```suggestion
      [containerd image store feature](https://docs.docker.com/storage/containerd/) enabled.
```

>     * If you are using a buildx instance, or using buildkitd directly, there is no need to enable the containerd snapshotter.

Reviewer: ```suggestion
    * If you are using a buildx instance, or using buildkitd directly, there is no need to enable the containerd image store.
```

looks like "containerd image store" is the marketing name

    > ensure that the credentials are configured in the default Docker config.json before running `copa patch`,
    > for example, via `sudo docker login -u <user> -p <password> <registry>`.

>     > **NOTE:** if you're scanning and patching an image that is local-only
>     > (i.e. built or tagged locally but not pushed to a registry), you are
>     > limited to using `docker`'s built-in buildkit service. See
>     > [Prerequisites][#prerequisites] for more information.

Reviewer: ```suggestion
> [!NOTE]   
> if you're scanning and patching an image that is local-only (i.e. built or tagged locally but not pushed to a registry), `copa` is limited to using `docker`'s built-in buildkit service, and must use [`containerd image store`](https://docs.docker.com/storage/containerd/) feature. See [Prerequisites][#prerequisites] for more information.
```

Reviewer: @pmengelbert i edited this while you were addressing this, let's make sure to add a reason

Author: Done, thanks.
